oauth2-server/src/main/java/com/clouway/oauth2/OAuth2Servlet.java

```diff
@@ -66,7 +67,7 @@ public void init(ServletConfig servletConfig) throws ServletException {
                                                                     config.identityFinder(),
                                                                     new IssueNewTokenActivity(
                                                                             config.tokens(),
-                                                                            new JjwtIdTokenFactory(config.keyStore())
+                                                                            idIdTokenFactory
```

Comment:
idId doesn't sound good. 

idTokenFactory

Reply:
I have overlooked this naming. Renamed to idTokenFactory